firmware/thruna/DCM/Src/Io/Io_EllipseImu.c

```diff
@@ -245,6 +328,44 @@ static void Io_EllipseImu_ProcessMsg_Status(const SbgBinaryLogData *log_data)
     sensor_data.status_data.com_status     = log_data->statusData.comStatus;
 }
 
+/*
+ * Process and save a relevant GPS Velocity information.
+ */
+static void Io_EllipseImu_ProcessMsg_GpsVel(const SbgBinaryLogData *log_data)
+{
+    // 0 means solution computed, 1-3 means an error occured (see binry log for more detail)
+    sensor_data.gps_data.gps1_velocity.status = sbgEComLogGpsVelGetStatus(log_data->gpsVelData.status);
```

Comment:
We should parse this in a new function, and then return it as an enum, so we can then broadcast it over CAN. If we can see something like `GPS_SOLUTION_COMPUTED` ON PCAN vs. just remembering 0 is the good state, that'd be great.